Cargo.toml

```diff
@@ -183,3 +183,4 @@ encrypt-cache-results = ["dep:chacha20poly1305", "dep:chacha20"]
 cec-cache-results = ["compress-cache-results", "encrypt-cache-results"]
 experimental-io-uring = ["actix-web/experimental-io-uring"]
 use-non-static-synonyms-search = ["thesaurus"]
+socks = ["reqwest/socks"]
```

Comment:
```suggestion
```

In accordance/agreement to your suggestion that we should not block this feature behind a compile flag then I would suggest removing this compile flag completely as we don't need to block such a small feature behind compile/build flag. 🙂